source/Marquee.js

```diff
@@ -232,6 +232,17 @@
 		* @method
 		* @private
 		*/
+		destroy: enyo.inherit(function (sup) {
+			return function () {
+				moon.Marquee.setMarqueeOnHoverControl(null);
```

Comment:
Shouldn't it only clear the marquee control if it is the one that was set? Otherwise it is clearing it unnecessarily.